website/docs/commands/run.md

```diff
@@ -146,8 +152,27 @@ Note that the [Scala Native requirements](https://scala-native.readthedocs.io/en
 scala-cli Hello.scala --native -S 2.13.6
 ```
 
+It is also possible to achieve it using `--platform` option:
+
+```bash
+scala-cli Hello.scala --platform native
+```
+
 We have a dedicated [Scala Native guide](../guides/scala-native.md) as well.
 
+## Platform
+
+The `--platform` option can be used to choose the platform, which should be used to compile and run application. Available platforms are:
+* JVM (`jvm`)
+* Scala.js (`scala.js` | `scala-js` | `scalajs` | `js`)
+* Scala Native (`scala-native` | `scalanative` | `native`)
+
+Please note that `--platform` option is prioritized over `--js` and `--native` options. For example, the following command will run using Scala Native:
+
+```bash
+scala-cli Hello.scala --js --platform scala-native
```

Comment:
I am thinking this command should actually throw an error, instead of prioritising `scala-native`
Similarly if `--js` and `--native` are passed together.